CHANGELOG

```diff
@@ -1,4 +1,5 @@
 0.64.0
+ - ref: add `get_contour_lines` method to `KernelDensityEstimator` class
```

Comment:
Please remember to keep the order in the changelog. The order should be `feat`, `fix`, `enh`, `setup`, `ref`, `tests`

Reply:
Ah ok, I thought the latest commit should be on the top. Thanks